app/models/case/base.rb

```diff
@@ -612,15 +612,31 @@ def already_late?
   end
 
   def num_days_late
-    days = ((date_responded.nil? ? Date.today : date_responded) - external_deadline).to_i
+    days = @deadline_calculator.class.days_late(
+      external_deadline, 
+      (date_responded.nil? ? Date.today : date_responded))
```

Comment:
This concept appears 5 times; wondering if it's a candidate for making into a wee method to name the concept and dry it up?

`(date_responded.nil? ? Date.today : date_responded)`